ts/Core/Tooltip.ts

```diff
@@ -1116,13 +1146,9 @@ class Tooltip {
                     });
 
                     // Set the stroke color of the box to reflect the point
-                    label.removeClass(/highcharts-color-[\d]+/g)
+                    label.removeClass(/.*/g)
```

Comment:
`removeClass` is not necessary. The [addClass](https://api.highcharts.com/class-reference/Highcharts.SVGElement#addClass) method has a parameter, `replace` that flushes previous classes.